HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -165,6 +165,13 @@ def customiseFor18429(process):
 
      return process
 
+# Updating FatjetJetProducer with fillDescriptions
+def customiseFor18832(process):
+    for producer in producers_by_type(process, "FastjetJetProducer"):
+        if not hasattr(producer, "doOutputJets"): producer.doOutputJets = cms.bool(False)
+        if not hasattr(producer, "sumRecHits"): producer.sumRecHits = cms.bool(False)
```

Comment:
ups, yeah sorry. I just tried to fix it quickly. I will run the runTheMatrix now with this changes before I commit the next change.